app/secrets.py

```diff
@@ -7,8 +9,12 @@
     "EQ_RABBITMQ_PASSWORD",
 ]
 
+SecretsType = Mapping[str, Mapping[str, str]]
```

Comment:
I think type alias should be able `REQUIRED_SECRETS`

Reply:
Don't think I quite know what you're getting at

Reply:
I shouldn't have read before posting 🙈 . Should say `I think type alias should be above REQUIRED_SECRETS`

Reply:
Ooooooooh haha I'll sort that 👍